airflow/bin/cli.py

```diff
@@ -441,6 +442,33 @@ def set_is_paused(is_paused, args):
     print("Dag: {}, paused: {}".format(args.dag_id, str(is_paused)))
 
 
+def show_dag(args):
+    dag = get_dag(args)
+    dot = render_dag(dag)
+    if args.save:
+        filename, _, fileformat = args.save.rpartition('.')
+        dot.render(filename=filename, format=fileformat, cleanup=True)
+        print("File {} saved".format(args.save))
+    elif args.imgcat:
+        data = dot.pipe(format='png')
+        try:
+            proc = subprocess.Popen("imgcat", stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+        except OSError as e:
+            if e.errno == errno.ENOENT:
+                raise AirflowException(
+                    "Failed to execute. Make sure the imgcat executables are on your systems \'PATH\'"
+                )
+            else:
+                raise
+        out, err = proc.communicate(data)
+        if out:
+            print(out.decode('utf-8'))
+        if err:
+            print(err.decode('utf-8'))
+    else:
+        print(dot.source)
```

Comment:
Given this also prints logging to stdout from airflow startup it might be worth adding a way to save the dot contents to a file. (Unless `dot.render(format="dot")` already works for that?)

Reply:
It works. I checked it.
```
airflow dags show example_gcp_vision_explicit_id --save a.dot
```

Reply:
After that, I thought about the problem that there are these logs.  I think airflow should be completely independent of CLI. Docker and kubernetes work similarly. This is important when we want Airflow to be completely serverless and secure. If I understand correctly, currently CLI always gives full access to Airflow. This is on my list of ideas.